jsk_naoqi_robot/pepper.rosinstall

```diff
@@ -2,4 +2,41 @@
     local-name: jsk_robot
     uri: https://github.com/jsk-ros-pkg/jsk_robot.git
     version: master
+# fix qt> 5 (https://github.com/ros-naoqi/naoqi_dashboard/pull/3)
+- git:
+    local-name: naoqi_dashboard
+    uri: https://github.com/kochigami/naoqi_dashboard
+    version: kochigami-develop
+# wait for 0.5.16 release
+- git:
+    local-name: nao_robot
+    uri: https://github.com/ros-naoqi/nao_robot
```

Comment:
In fact, I also use kochigami-develop for `nao_robot` package.
I'll consider whether we really need kochigami-develop or not.

Reply:
the difference is https://github.com/ros-naoqi/nao_robot/compare/master...kochigami:kochigami-develop

[modify nao_speech (delete speech topic)](https://github.com/ros-naoqi/nao_robot/commit/dff29823dc5c1c37972cc1a842cf16da506f0591)
[delete naoqi_bridge dependency and support name space](https://github.com/ros-naoqi/nao_robot/commit/dab1c3e719ea1947a35c58bbf9dccbad5762c720)